README.md

```diff
@@ -1,5 +1,58 @@
+| STATUS_LIST_VC_ID      | VC id of status list credential of base wallet                                   | 8a6c7486-1e1f-4555-bdd2-1a178182651e |
+| TOKEN_EXPIRY_TIME      | JWT(STS, VC and VP) expiry time in minutes                                       | 5                                    |
+| PORTAL_WAIT_TIME       | Wait time before we push did document to portal after wallet creation in seconds | 60                                   |
+| PORTAL_HOST            | Host of port backend application                                                 |                                      |
+| PORTAL_CLIENT_ID       | Keycloak client_id to access portal API                                          |                                      |
+| PORTAL_CLIENT_SECRET   | keycloak client_secret to access portal API                                      |                                      |
+| PORTAL_REALM           | keycloak realm                                                                   |                                      |
+| PORTAL_AUTH_SERVER_URL | Authentication server(keycloak)                                                  |                                      |
+| APP_LOG_LEVEL          | Log level of application                                                         | DEBUG                                |
+
+2. Run application using gradle
+
+`./gradlew clean bootrun`
+
+3. After successful running orf the application, you can access swagger
+   on  ``http://localhost:8080/ui/swagger-ui/index.html``
+
+### Documentation
+
+Detailed documentation can be found [here](docs%2FREADME.md)
+
+### Important notes and limitation of application
+
+1. Simple Java ``Map`` is used to store keypair, VC and VP of wallet to avoid any further complexity. Please
+   refer [MemoryStorage.java](src%2Fmain%2Fjava%2Forg%2Feclipse%2Ftractusx%2Fwallet%2Fstub%2Fstorage%2FMemoryStorage.java)
+2. This application will create same key for given BPN on given environment. Please
+   refer [DeterministicECKeyPairGenerator.java](src%2Fmain%2Fjava%2Forg%2Feclipse%2Ftractusx%2Fwallet%2Fstub%2Futils%2FDeterministicECKeyPairGenerator.java)
+3. If a wallet is not created at any point of request, application will create a new wallet at runtime
+4. Application will issue new VC during ``/presentations/query`` API if not issued previously
+5. Application will not give error if request VC is not already issue
+6. There is no separate IDP for this wallet application. OAuth token creation API is part of this application only
+7. We are not validating ``client_secret`` while creating OAuth token and  ``client_id`` will be BPN
+8. Negative scenarios are not covered
+9. ``jti`` claim is not validated
+10. No actual revocation of verifiable credentials
+11. All stored credentials will be lost on restart of the application
+12. JWTs are printed with debug log level for debugging purposes
```

Comment:
the notice for docker images should be linked, see example:https://github.com/eclipse-tractusx/portal-backend?tab=readme-ov-file#notice-for-docker-image

Reply:
Adressed with: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/2/commits/3d0d3bb98b833e020fdb59e7a986a2e62b885018